### PR TITLE
Tests for Swedish translation

### PR DESCRIPTION
**Pull request by bertfrees**
Includes:
- [ ] uncontracted braille
- [ ] 8-dot braille
- [ ] contracted braille
- [ ] emphasis indicators

Addresses requirements:
- [ ] [4.3:34](http://snaekobbi.github.io/requirements#4.3:34)
- [ ] [4.3:35](http://snaekobbi.github.io/requirements#4.3:35)
- [ ] [4.3:37](http://snaekobbi.github.io/requirements#4.3:37)
- [ ] [4.3:38](http://snaekobbi.github.io/requirements#4.3:38)
- [ ] [4.3:39](http://snaekobbi.github.io/requirements#4.3:39)

### CODE REVIEW

File: src/resources/34/data.xml [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+		<input>Hör upp!</input>
+	</entry>
+
+	<entry>
+		<expect>⠠⠓⠕⠝⠀⠎⠁⠒⠀⠠⠠⠙⠝⠒⠎⠀⠗⠑⠙⠁⠅⠞⠊⠕⠝⠀⠜⠗⠀⠎⠞⠕⠗⠄</expect>
+		<input>Hon sa: DN:s redaktion är stor.</input>
+	</entry>
+
+	<entry>
+		<expect>⠠⠎⠅⠊⠇⠇⠝⠁⠙⠑⠝⠀⠍⠑⠇⠇⠁⠝⠀⠁⠗⠃⠑⠞⠎-⠀⠕⠉⠓⠀⠧⠊⠇⠕⠙⠁⠛⠁⠗⠀⠃⠇⠑⠧⠀⠍⠊⠝⠙⠗⠑⠀⠎⠅⠁⠗⠏⠆⠀⠓⠕⠝⠀⠅⠥⠝⠙⠑⠀⠞⠊⠇⠇⠡⠞⠁⠀⠎⠊⠛⠀⠧⠊⠇⠕⠙⠁⠛⠁⠗⠀⠍⠊⠞⠞⠀⠊⠀⠧⠑⠉⠅⠁⠝⠄</expect>
```
Comment (bertfrees) [inline src/resources/34/data.xml]: Why are there hyphens in the braille?

**Comment by bertfrees** [comment]
The Dotify test works now. The liblouis translation is still wrong.

**Comment by bertfrees** [comment]
There is currently no definition of 8-dot braille. Need to find out whether defining and/or implementing are in scope. If yes, what is the priority.

**Comment by bertfrees** [comment]
To do:
- [ ] Move contracted to priority 2?

**Comment by joeha480** [comment]
Priority 2 for this sprint, but probably not for the project

**Pull request closed without merging**